tests/test_kendall_tau_distance.py

```diff
@@ -22,8 +29,15 @@ def test_pairwise_distance_matrix():
     r1 = [0.05, 0.2, 0.7, 0.05]
     r2 = [0.23, 0.24, 0.26, 0.27]
     r3 = [0.22, 0.24, 0.26, 0.28]
-    rankings = [r1, r2, r3]
+    rankings = np.array([r1, r2, r3])
     D = pairwise_distance_matrix(rankings)
+
+    # Testing dask
+    client = Client()
+    D_dask = pairwise_distance_matrix(da.from_array(rankings))
```

Comment:
Does this function not return a dask array? If so you would need to do a compute or at least move the `client.close()` statement after the assert.